### PR TITLE
fix: work of next button#554

### LINKED ISSUE #554: [MAJOR] How the Next document button works
If the annotator has tasks from different jobs and there is a change of priorities (ex. from UAT 12.6 job with 643 files to UAT 12.6 primary job with 60 files), then he can't start from the "last document" of the new job, because then there is no next document for the Next button and the annotator needs to go to Dashboard for a new task
image
image
The workaround only:
Since there is no filtering by Job, it needs to manually scroll through the pages to find the start of the other job.
image

### PR DESCRIPTION
**Pull request by serereg**
Closes #554

### CODE REVIEW

**Review by spiridonovfed — APPROVED** [review]

**Comment by spiridonovfed** [comment]
Code changes are approved, but please correct tests

File: annotation/tests/test_tasks_crud_cr.py [hunk 1]
```diff
@@ -1449,20 +1505,22 @@ def test_search_tasks_pagination(
...
         (
             "user_id",
             "eq",
             "9eace50e-613e-4352-b287-85fd91c88b51",
             list(range(3, 7)),
         ),
-        ("is_validation", "ne", False, [5, 6, 20, 23]),
-        ("is_validation", "ne", False, [5, 6, 20, 23]),
+        # FIXME
+        pytest.param("is_validation", "ne", False, [5, 6, 20, 23], marks=pytest.mark.xfail(reason="fix the test")),
```
Comment (serereg) [inline annotation/tests/test_tasks_crud_cr.py:1517]: It relates to 
https://github.com/epam/badgerdoc/issues/121

File: docker-compose-dev.yaml [hunk 2]
```diff
@@ -95,6 +95,9 @@ services:
     depends_on:
       badgerdoc-postgresql:
         condition: service_healthy
+    volumes:
+      - ./annotation/:/opt/annotation/
+    command: bash -c "alembic upgrade head && uvicorn annotation.main:app --host 0.0.0.0 --port 8080 --reload"
```
Comment (cakeinsauce) [inline docker-compose-dev.yaml:100]: We have this line in Dockerfile https://github.com/epam/badgerdoc/blob/main/annotation/Dockerfile#L20
Just specify build target
```yml
    build:
      context: .
      target: build-dev
```

Comment (cakeinsauce) [reply docker-compose-dev.yaml:100]: And do you think we need to mount the volume? We copy all necessary files during build in the Dockerfile

Comment (serereg) [reply docker-compose-dev.yaml]: Done

File: annotation/tests/test_tasks_crud_cr.py (hunk 1 above)
Comment (serereg) [reply annotation/tests/test_tasks_crud_cr.py:1518]: Done

**Pull request merged**